### DISCUSSION TITLE
Keyhole mounting

### DISCUSSION THREAD

**Discussion by ctvw** [aderusha/DDD-Printable-Wall-Control-System#66] [Ideas]
Looking for a low profile solution for anything that has a keyhole mount.

**Comment by mcharach** [comment]
I agonized over this, just asking this exact question last week.  

This is an excellent solution.  Its cheap semipermanent, and you will need to add some M4 screws with the right sized head for your charger, keyhole, or printed mount.  I built a charging wall and had many items that needed to be screwed or keyhole mounted.  The threaded nuts work great.  Thanks to whoever it was in the group that recommended this solution.  Pictures to follow once I am done with the wire management. 

https://a.co/d/8msAOrC

**Reply by ctvw** [reply]
@mcharach - and those don't damage the coating on the panels? How hard are they to remove?

**Comment by aderusha** [comment]
That's a great solution!  I've been using heat-set inserts for the same purpose.  Just take any of the [Spacer Blank](https://github.com/aderusha/DDD-Printable-Wall-Control-System/tree/main/Centerpieces/Spacer_blank) models, and put a hole where you want your keyhole to be.  Put an insert (the rivnut style linked above would be great) where you want that screw to go, now you have a machine thread screw mounted securely to your wall!

**Comment by razorfrog** [comment]
What about this? https://www.printables.com/model/206047-wall-control-sturdy-hanger-w-m4m5-screw

**Comment by mcharach** [comment]
I put over a dozen of them on.  None seemed to damage the finish.  Taking them off would be messy maybe  you could compress one with some needle nose pliers, or worst case scenario drill it out.  

But my thinking is that I still have hundreds of other holes that I could use pegs on and the wall control slits are still there, so why would you even bother removing them even if you switched things around.
